jaxrs-service/src/main/java/com/quorum/tessera/p2p/PartyInfoResource.java

```diff
@@ -56,8 +54,45 @@ public Response partyInfo(@ApiParam(required = true) final byte[] payload) {
 
         final StreamingOutput streamingOutput = out -> out.write(encoded);
 
-        return Response.status(Response.Status.OK)
-            .entity(streamingOutput)
-            .build();
+        return Response.status(Response.Status.OK).entity(streamingOutput).build();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Fetch network/peer information", produces = "public list of peers/publickey mappings")
+    @ApiResponses({@ApiResponse(code = 200, message = "Peer/Network information", response = PartyInfo.class)})
+    public Response getPartyInfo() {
```

Comment:
Is this a good time to consider adding a "monitoring" app (where the new service is exposed)? Although it is easy to test in most scenarios (where nobody bothers with P2P app encryption) when the P2P app is secured it may prove harder to invoke with tools like curl (where a java key store may not be immediately usable).